### DISCUSSION TITLE
Lombiq Helpful Libraries with Docker  error.

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#16612] [Q&A]
I running into an issue trying to use the Lombiq libraries in my orchard project.  

Here's the error:

124.4   Lombiq.Hosting.Tenants.Management -> /app/Modules/Lombiq.Hosting.Tenants/Lombiq.Hosting.Tenants.Management/bin/Release/net8.0/Lombiq.Hosting.Tenants.Management.dll
124.5   Lombiq.Hosting.Tenants.Admin.Login -> /app/Modules/Lombiq.Hosting.Tenants/Lombiq.Hosting.Tenants.Admin.Login/bin/Release/net8.0/Lombiq.Hosting.Tenants.Admin.Login.dll
125.0   Lombiq.Hosting.BuildVersionDisplay -> /app/Modules/Lombiq.Hosting.BuildVersionDisplay/Lombiq.Hosting.BuildVersionDisplay/bin/Release/net8.0/Lombiq.Hosting.BuildVersionDisplay.dll
125.0   Lombiq.Hosting.Tenants.Maintenance -> /app/Modules/Lombiq.Hosting.Tenants/Lombiq.Hosting.Tenants.Maintenance/bin/Release/net8.0/Lombiq.Hosting.Tenants.Maintenance.dll
125.3   /usr/bin/sh: 2: /tmp/MSBuildTemproot/tmp61a9b98505954636805ad890cd949f04.exec.cmd: corepack: not found
125.4   /usr/bin/sh: 2: /tmp/MSBuildTemproot/tmp4c53f2327ed243e0a5925d0852e0beea.exec.cmd: pnpm: not found
125.4   /usr/bin/sh: 2: /tmp/MSBuildTemproot/tmpd1a4ef13f930494e8fe4fce6b52fcb1e.exec.cmd: pnpm: not found
125.4   /usr/bin/sh: 2: /tmp/MSBuildTemproot/tmpc6284ed046ee47208666e9f09f355da2.exec.cmd: pnpm: not found
125.5   /usr/bin/sh: 2: /tmp/MSBuildTemproot/tmp8383f8fc44564dc097c291428d2477fc.exec.cmd: pnpm: not found
125.5   /usr/bin/sh: 2: /tmp/MSBuildTemproot/tmpfcfa38c232774e568207c4128881ba95.exec.cmd: pnpm: not found
125.5 /app/Utilities/Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions/build/RetryCommand.targets(33,5): error MSB3073: The command "pnpm install --prod --prefer-offline" exited with code 127.

Here's what I've tried in my Dockerfile:

RUN apt update -y && apt upgrade -y && apt install nodejs -y && apt install npm -y
RUN npm install --global pnpm
RUN pnpm config set store-dir /root/.local/share/pnpm/store/v3 --global
RUN pnpm i -g

Here's what I've tried in my docker-compose:

entrypoint: ["/bin/sh","-c"]
command:
- |
   apt update -y 
   apt upgrade -y
   apt install nodejs -y 
   apt install npm -y
   npm install --global pnpm
   pnpm config set store-dir /root/.local/share/pnpm/ --global
   pnpm install
   npm install -g corepack
   corepack enable

My error probably comes from a lack of understanding on my part.  Still fairly new to all of this.

**Comment by hishamco** [comment]
I suggest filing an issue in [HelpFull Libraries repo ](https://github.com/Lombiq/Helpful-Libraries )
